go.mod

```diff
@@ -0,0 +1,44 @@
+module github.com/rimusz/monocular
```

Comment:
guess this should be github.com/helm/monocular?

Reply:
yes, autogeneration, will fix it up

Reply:
done